### PR TITLE
Addition of the Cornu horn for Vexillarius.

### PR DESCRIPTION
**Pull request by DIamonnd**
## Description

Addition of a roman cornu horn to the Legion's Vexillarius. This item works as a megaphone with remplaced icons (both item and inhand), sound and other flavor variables.
## Motivation and Context

To allow the Vexillarius to fulfil their role of being the flag bearer of troops with more ease rather than just being another legionnare with a fancier weapon.
## How Has This Been Tested?

Hosted server and joined as Vexilalrius, icon, sound, spam check and spawn seem to be working just fine.
## Screenshots (if appropriate):
imagen

### CODE REVIEW

**Comment by Kimochiiiiiiiiiiiiii** [comment]
How many sounds does it make?

**Comment by DIamonnd** [comment]
It plays a horn sound that is characteristic for the object. I believe its possible to make it work as instruments do. But would be somewhat weird.

**Comment by Kimochiiiiiiiiiiiiii** [comment]
**"YOU GUYS KNOW BUSH DIFF ISIS?? "**
Imagine Undertale once that horn

**Comment by Raze500** [comment]
**Why my Peeepee hard?**

**Comment by UndeadNappist** [comment]
You can now just go in and define all of /job/Caesarslegion in the datum to ensure that only those of the legion-faction type job can hear something ; anchor it to a trait, or a proc, or whatever your preference is. Either way, it's now a lot easier for you to implement.

File: code/game/objects/items/devices/megaphone.dm [hunk 1]
```diff
@@ -48,4 +48,21 @@
...
+	icon_state = "cornu"
+	item_state = "cornu"
+	lefthand_file = 'icons/mob/inhands/equipment/horns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/horns_righthand.dmi'
+
+/obj/item/megaphone/cornu/get_held_item_speechspans(mob/living/carbon/user)
+	if(spamcheck > world.time)
+		to_chat(user, "<span class='warning'>\The [src] is not to be overused in battle!</span>")
+	else
+		playsound(loc, 'sound/items/airhorn.ogg', 100, 0, 1)
```
Comment (rhicora) [inline code/game/objects/items/devices/megaphone.dm]: Go ahead and upload a new sound that is more characteristic of this cornu.

We're not going with airhorn.ogg.

**Pull request merged**